thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/completeness/checker/DataCompletenessUtils.java

```diff
@@ -261,13 +261,4 @@ public static DateTimeFormatter getDateTimeFormatterForDataset(TimeSpec timeSpec
     return bucketNameToCountStar;
   }
 
-  public static double getPercentCompleteness(PercentCompletenessFunctionInput input) {
```

Comment:
This whole class could be deleted.

Reply:
This class is used in `Wo4WAvgDataCompletenessAlgorithm`. We need to drop both of them if we want to drop this. `Wo4WAvgDataCompletenessAlgorithm` cannot be dropped due to the issue below.

Reply:
We should remove it and also remove getter/setter.
That should be safe.

Reply:
I remove `Wo4WAvgDataCompletenessAlgorithm` completely and also the field `DatasetConfigBean.dataCompletenessAlgorithm` from `DatasetConfigBean`.